taskfiles/tests/integration.yaml

```diff
@@ -0,0 +1,27 @@
+version: "3"
+
+vars:
+  CLP_BUILD_DIR: "{{.G_BUILD_DIR}}"
+  CLP_CORE_BINS_DIR: "{{.G_CORE_COMPONENT_BUILD_DIR}}"
+  CLP_PACKAGE_DIR: "{{.G_PACKAGE_BUILD_DIR}}"
+  G_INTEGRATION_TESTS_DIR: "{{.ROOT_DIR}}/integration-tests"
+
+tasks:
+  default:
+    deps:
+      - task: "core"
+
+  cache-clear:
+    dir: "{{.G_INTEGRATION_TESTS_DIR}}"
+    cmds:
+      - >-
+        uv run python -m pytest --cache-clear --collect-only --override-ini addopts="" --quiet
+        > /dev/null
+
+  core:
+    deps:
+      - task: "::package"
+    dir: "{{.G_INTEGRATION_TESTS_DIR}}"
+    cmds:
+      - >-
+        uv run python -m pytest -m "core"
```

Comment:
Why not `uv tool run pytest -m core`? Ditto above and in the docs.

Reply:
I think in my `.pytest.ini` I uses `env` config which would require `pytest-env`. Using `uv tool  run pytest` won't bring in `pytest-env` or other tool dependencies that `pytest` may use.